ZUGFeRD/InvoiceDescriptor22UBLWriter.cs

```diff
@@ -371,9 +379,12 @@ public override void Save(InvoiceDescriptor descriptor, Stream stream, ZUGFeRDFo
                 Writer.WriteElementString("cbc:Description", tradeLineItem.Description);
                 Writer.WriteElementString("cbc:Name", tradeLineItem.Name);
 
-                Writer.WriteStartElement("cac:SellersItemIdentification");
-                Writer.WriteElementString("cbc:ID", tradeLineItem.SellerAssignedID);
-                Writer.WriteEndElement(); //!SellersItemIdentification
+                if (tradeLineItem.SellerAssignedID != null && !string.IsNullOrWhiteSpace(tradeLineItem.SellerAssignedID))
```

Comment:
imho you can skip tradeLineItem.SellerAssignedID != null, that is checked by IsNullOrWhiteSpace as well :)